system/doc/design_principles/events.md

```diff
@@ -21,8 +21,8 @@ limitations under the License.
 
 [](){: #gen_event }
 
-This section is to be read with the `m:gen_event` manual page in STDLIB, where
-all interface functions and callback functions are described in detail.
+When reading this section, reference module `m:gen_event` in STDLIB for detailed
+explanations of all interface and callback functions.
```

Comment:
```suggestion
When reading this section, see module `m:gen_event` in STDLIB for detailed
explanations of all interface and callback functions.
```
Or maybe "see the reference manual for..."? With just "reference" (as a verb), this took me a moment to understand and realize it was (probably) not a typo :sweat_smile:

Reply:
Yes, it is not a typo, but I didn't really like it after some consideration, so I used another phrasing in other places. I forgot to change this instance. Here's how I will change it:

> It is recommended to read this section alongside `m:gen_event` in STDLIB.